src/modules/cpu/kernel/tensor_sum.hpp

```diff
@@ -0,0 +1,842 @@
+#if __AVX2__
+            __m256i psum = _mm256_setzero_si256();
+#endif
+            for(int i = 0; i < roi.xywhROI.roiHeight; i++)
+            {
+                Rpp8u *srcPtrTemp;
+                srcPtrTemp = srcPtrRow;
+
+                int vectorLoopCount = 0;
+                for (; vectorLoopCount < alignedLength; vectorLoopCount += vectorIncrementPerChannel)
+                {
+#if __AVX2__
+                    __m256i p1[2];
+                    rpp_simd_load(rpp_load16_u8_to_u32_avx, srcPtrTemp, p1);
+                    compute_sum_16_host(p1, &psum);
+#endif
+                    srcPtrTemp += vectorIncrementPerChannel;
+                }
+                for (; vectorLoopCount < bufferLength; vectorLoopCount++)
+                {
+                    sum += (Rpp32u)(*srcPtrTemp);
+                    srcPtrTemp++;
+                }
+                srcPtrRow += srcDescPtr->strides.hStride;
+            }
+#if __AVX2__
+            rpp_simd_store(rpp_store8_u32_to_u32_avx, sumAvx, &psum);
+#endif
+            for(int i = 0; i < 4; i++)
+                sum += (sumAvx[i] + sumAvx[i + 4]);
+
+            tensorSumArr[batchCount] = (Rpp64u)sum;
+        }
+
+        // Tensor Sum without fused output-layout toggle 3 channel (NCHW)
+        else if ((srcDescPtr->c == 3) && (srcDescPtr->layout == RpptLayout::NCHW))
+        {
+            Rpp64u sum;
+            Rpp32u sumR = 0, sumG = 0, sumB = 0;
+            Rpp32u sumAvxR[8] = {0};
+            Rpp32u sumAvxG[8] = {0};
+            Rpp32u sumAvxB[8] = {0};
+
+            Rpp8u *srcPtrRowR, *srcPtrRowG, *srcPtrRowB;
+            srcPtrRowR = srcPtrChannel;
+            srcPtrRowG = srcPtrRowR + srcDescPtr->strides.cStride;
+            srcPtrRowB = srcPtrRowG + srcDescPtr->strides.cStride;
+#if __AVX2__
+            __m256i psumR = _mm256_setzero_si256();
+            __m256i psumG = _mm256_setzero_si256();
+            __m256i psumB = _mm256_setzero_si256();
+#endif
+            for(int i = 0; i < roi.xywhROI.roiHeight; i++)
+            {
+                Rpp8u *srcPtrTempR, *srcPtrTempG, *srcPtrTempB;
+                srcPtrTempR = srcPtrRowR;
+                srcPtrTempG = srcPtrRowG;
+                srcPtrTempB = srcPtrRowB;
+
+                int vectorLoopCount = 0;
+                for (; vectorLoopCount < alignedLength; vectorLoopCount += vectorIncrementPerChannel)
+                {
+#if __AVX2__
+                    __m256i p[6];
+                    rpp_simd_load(rpp_load48_u8pln3_to_u32pln3_avx, srcPtrTempR, srcPtrTempG, srcPtrTempB, p);
+                    compute_sum_48_host(p, &psumR, &psumG, &psumB);
+#endif
+                    srcPtrTempR += vectorIncrementPerChannel;
+                    srcPtrTempG += vectorIncrementPerChannel;
+                    srcPtrTempB += vectorIncrementPerChannel;
+                }
+                for (; vectorLoopCount < bufferLength; vectorLoopCount++)
+                {
+                    sumR += (Rpp32u)(*srcPtrTempR);
```

Comment:
please combine srcPtrTempR++ in this line and get rid of lines 120 to 122

Reply:
Modified code accordingly.

Reply:
Incorporated this change

Reply:
I don't see anything changed here. Please commit all your changes based on review comments

Reply:
@rrawther Not sure what's going on with the Github UI. These changes have been addressed though. 

They show up when you open the commit or the file, but not when you click on the comment to verify.
The changes are showing even if we go again to "Files Changed" section or click on "View File".
https://github.com/GPUOpen-ProfessionalCompute-Libraries/rpp/blob/97f388f1b4df575c70b3b4f01aeb066fbf67964f/src/modules/cpu/kernel/tensor_sum.hpp#L112-L116